docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -16,8 +16,8 @@ We'll do our best to keep things easy to follow, and as sequential as possible s
 
 ## Why you should migrate
 
-This documentation page covers the *how* of migrating from v4 to v5.
-The *why* is covered in the [release blog post on Medium](https://medium.com/material-ui/material-ui-v4-is-out-4b7587d1e701).
+This documentation page covers the _how_ of migrating from v4 to v5.
```

Comment:
I guess it will help for formating all the markdown with prettier later down the road.

Reply:
Just tried. I'll share some points why we shouldn't :upside_down_face:

Reply:
Now might be the best time to do it, assuming a lot of the translations will change anyway;

Reply:
Agreed! That's why I went down that rabbit hole again.

I thought there were some actual technical limitations but it looks good now. Still need to check each change.